test/routes/testSoldierRouts.js

```diff
@@ -0,0 +1,97 @@
+const expect = require('chai').expect;
+const assert = require('assert');
+const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+const testServer = require("../../start.js").server;
+const MongoClient = require('mongodb').MongoClient;
+
+const DBurl = 'mongodb://localhost:27017';
+const dbName = 'myproject';
+const solId = "tt8145643";
+const solName = "lior";
+const rank = "kama";
+const limitations = ["yeshiva"]
+const standartSol = JSON.stringify({
+  "id": solId,
+  "name": solName,
+  "rank": rank,
+  "limitations": limitations
+})
+const url = 'http://localhost:3000/soldiers';
+
+describe("Soldiers tests", function () {
+
+  before("Initialaizing db", function (doneBefore) {
+    MongoClient.connect(DBurl, function (dbConnectErr, client) {
+      if (dbConnectErr) return doneBefore(dbConnectErr);
+      testServer.dbClient = client;
+      testServer.collection = client.db(dbName).collection('Soldiers')
+    });
+    doneBefore();
+  });
+
+  afterEach("db cleaning", function (doneAfterEach) {
+    testServer.collection.deleteMany({}, (deleteError) => {
+      if(deleteError) return doneAfterEach(deleteError);
+    })
+    doneAfterEach();
+  });
+
+  after("Closing the server", function (doneAfter) {
```

Comment:
If you're making this sync and not async, if I remember correctly you can just omit the callback from the function you write here and it'll just run as sync.